### PR TITLE
New Overview Page + Guides Page

### PR DESCRIPTION
**Pull request by ndneighbor**
A more welcoming page for the docs experience.

### CODE REVIEW

File: src/pages/develop/projects.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A Railway project contains all of your plugins, environments, and deployments. I
 
 All projects feature
 
-- 100 GB outbound network bandwidth
+- 80 GB outbound network bandwidth
```
Comment (coffee-cup) [inline src/pages/develop/projects.md:9]: Don't think this was supposed to be changed

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,82 +1,102 @@
...
-            </h3>
-            <p>
-              Each page footer contains an "Edit on GitHub Link". Change the
-              markdown, make a pull request, and we'll merge it! Deploys will
-              happen automagically cause the docs are hosted on Railway.
-              <br />
-              <Link href="https://github.com/railwayapp/docs">Doc Source</Link>
-            </p>
-          </section>
+        <div tw="mt-8 grid grid-cols-1 gap-8 md:grid-cols-2">
```
Comment (coffee-cup) [inline src/pages/index.tsx:47]: nit/suggestion: I think the gap should be a bit smaller on mobile.

image
```suggestion
        <div tw="mt-8 grid grid-cols-1 gap-4 md:gap-8 md:grid-cols-2">
```

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,82 +1,102 @@
...
+              <OverviewLinkText>Tutorials on common developer scenarios.</OverviewLinkText>
+            </OverviewLinkTextbox>
+          </OverviewLink>
+        </div>
+        <div tw="grid grid-cols-1 mt-14 items-center md:grid-cols-8">
+          <div tw="bg-gray-100 rounded-md col-span-5 p-6">
+            <div tw="flex text-base font-semibold mb-4"><GitHub tw="mr-2" /> Contributing</div>
+            <div>
+              <div>Each page footer contains an "Edit on GitHub" Link. Make a pull request, and we'll merge it!</div>
+              <Link tw="flex text-pink-800 items-center mt-2" href="https://github.com/railwayapp/docs">Docs Repository <ArrowRight /></Link>
```
Comment (coffee-cup) [inline src/pages/index.tsx:90]: This should have a hover style

image

**Review by coffee-cup — APPROVED** [review]

File: src/pages/develop/projects.md (hunk 1 above)
Comment (ndneighbor) [reply src/pages/develop/projects.md:9]: That was a mistake on my part. Will fix.

**Pull request merged**